### PR TITLE
Fixed the integration(unit) tests

### PR DESCRIPTION
**Pull request by kennywlo**
Fixed several issues with the integration(unit) tests. The test datasets, made of ad hoc image cutouts, been removed, since it's very high maintenance and no longer useful. Now the tests will require presence of /datasets/...

### CODE REVIEW

File: python/lsst/dax/imgserv/version.py [hunk 1]
```diff
@@ -1,4 +0,0 @@
-from pkg_resources import get_distribution
```
Comment (cbanek) [inline python/lsst/dax/imgserv/version.py]: Please don't delete this file.  I need this for things to work with the new packaging scheme.

File: python/lsst/dax/imgserv/cli_v1.py [hunk 2]
```diff
@@ -42,8 +42,24 @@
 from lsst.dax.imgserv.hashutil import Hasher
 from lsst.dax.imgserv.jsonutil import flatten_json
 
+try:
+    from ConfigParser import RawConfigParser, NoSectionError
+except ImportError:
+    from configparser import RawConfigParser, NoSectionError
```
Comment (cbanek) [inline python/lsst/dax/imgserv/cli_v1.py]: What's with this?  Seems like a capitalization difference?  Is this across versions?

File: python/lsst/dax/imgserv/cli_v1.py [hunk 3]
```diff
@@ -42,8 +42,24 @@
...
+
+# Initialize configuration
+webserv_parser = RawConfigParser()
+webserv_parser.optionxform = str
+
+with open(os.path.expanduser(defaults_file)) as cfg:
+    webserv_parser.readfp(cfg, defaults_file)
+
+webserv_config = dict(webserv_parser.items("webserv"))
+imgserv_meta_url = webserv_config.get("dax.imgserv.meta.url")
```
Comment (cbanek) [inline python/lsst/dax/imgserv/cli_v1.py]: Is this related to the tests?  I'm not sure I get why this is changing?  Or perhaps this is something else that was different?

File: python/lsst/dax/imgserv/cli_v1.py (hunk 2 above)
Comment (timj) [reply python/lsst/dax/imgserv/cli_v1.py]: This is python2 compatibility shim and should be removed.

File: python/lsst/dax/imgserv/cli_v1.py (hunk 3 above)
Comment (cbanek) [reply python/lsst/dax/imgserv/cli_v1.py]: Ah gotcha, makes sense.

File: python/lsst/dax/imgserv/version.py (hunk 1 above)
Comment (cbanek) [reply python/lsst/dax/imgserv/version.py]: That file is not the one automatically generated by scons, that's the one I wrote.  If you run scons, it will generate one on top of it though (rewrite it) with different content.

**Comment by cbanek** [comment]
Not sure what's going on with that version.py, it seems to have all the scons stuff in it now but it's edited.  Please replace it with the original version on master:

from pkg_resources import get_distribution

__version__ = get_distribution('imgserv').version
__all__ = ['__version__']

**Comment by cbanek** [comment]
Also just to note, without the data checked into  this repo, we won't be able to run these tests outside of NCSA because we won't have /datasets access, right?

**Comment by kennywlo** [comment]
Besides /datasets, these integration tests require login access to the meta data (via metaserv/dbserv) on PDAC. It would not be a trivial effort to decouple from all these, for a suitable test harness.

**Comment by cbanek** [comment]
Sorry what is "login access to the image metadata" mean?  Does that mean that it needs to be able to contact metaserv API endpoints?

**Comment by kennywlo** [comment]
It has to do with how the butler (imgserv key dependency) works - essentially the image metadata (e.g. FITS headers) are processed into the DB (qsmaster), which allows searches, e.g RaDec, to locate the actually image, return the data (image) ids that butler can then fetch from /datasets file system.  More on that later...
